docs/oauth2-oidc/custom-login-consent/flow.mdx

```diff
@@ -117,17 +118,82 @@ The way you collect the consent information from the end user is up to you. In m
 </form>
 ```
 
+### Skipping consent for trusted clients
+
+When you create an OAuth2 client in your Ory Network project, you can configure it to skip the "consent screen", which is the
+screen where the user must explicitly agree to giving the client access to access their data and perform operations on their
+behalf.
+
+This approach is useful when you're working with trusted, first-party clients that don't require access to sensitive data. When
```

Comment:
It's not about the sensitivity of the data, more about if the client is managed by the same entity as the whole Ory Network project. If customers deploy OAuth2 just to get the JWTs, but have their own OAuth2 client, that's a first-party client.